docs/manual/developer/06_contributing_with_content.md

```diff
@@ -1773,6 +1773,7 @@ The remediations add the `Defaults` option  to `/etc/sudoers` file.
 -   Parameters:
 
     - **option** - name of sudo `Defaults` option to enable.
+    - **option_regex_suffix** - suffix to the pattern-match to use after **option**; defaults to `=(\w+)`.
```

Comment:
The `\b' is missing in the default `option_regex_suffix`.

Reply:
Ah, d'oh thanks for the catch :)